src/pybind/mgr/orchestrator.py

```diff
@@ -183,6 +186,23 @@ def describe_service(self, service_type=None, service_id=None, node_name=None):
         """
         raise NotImplementedError()
 
+    def service_action(self, action, service_type, service_name=None, service_id=None):
```

Comment:
it's not clear to me the difference between `service_type` and `service_name`. Could you give a real example of how these two parameter are used with different values?

Reply:
AIUI, type would be something like "nfs" or "mds". The bigger distinction is between "name" and "id" here, and I think that's more or less described in the delta below.